script/Ciel/Day 14/14_01A_CIEL14A_6.txt

```diff
@@ -256,7 +278,8 @@ I put pressure into the hand holding the knife
 [sha:767aff02d7c453b588aa835c668895b73b34b9d4]{
 -- Page 222, Offset 38296.
 -- スウ、と音もなく感触さえなく、ナイフの刃が体に透る。
-Quietly, and without feeling, the blade passes through my body.
+It doesn't make a sound.
+ I don't even feel it as the blade sinks into my body.
 }
 [sha:733241b6638265f0c33c2a69888c37b15e8a6147]{
 -- Page 222, Offset 38297. Mods: @n.
```

Comment:
These make me think that maybe 38282 should be 'That's all', but I'm not sure how much of a difference it makes. It works here better for sure, but...